integration-mc1-15/src/main/kotlin/org/eln2/ModItems.kt

```diff
@@ -1,11 +1,14 @@
 package org.eln2
 
 import net.minecraft.item.Item
+import net.minecraft.item.ItemGroup
 import org.eln2.items.MultimeterItem
+import org.eln2.items.OreChunks
 
 /**
  * Items added here are automatically registered.
  */
 enum class ModItems(val items: Item) {
-	MULTIMETER(MultimeterItem())
+	MULTIMETER(MultimeterItem()),
+	ORE_CHUNKS_COPPER(OreChunks())
```

Comment:
Bookshelf automatically creates an item for each registered block, unless overridden, which we don't currently. You shouldn't need to add one here, and if you do, you'll need to change the block registration to support passing in the item.

The way you're doing things here, you'll get a separate block-item (created by Bookshelf) from the item dropped by the ore when broken. That seems to be what you want, but it's worth a comment.

Reply:
So I know there are item blocks, but the point of the code here is that ore chunks are *not* the original ore block. The problem with Minecraft `ItemBlock`'s is that you can place them again, eg, if you break an iron ore block, you can place it again. I don't want that, and later, we may even make it so that these piles of dust have NBT/internal data concerning what is actually in the particular sample of ore; eg, 30% iron (ii) oxide, 60% stone, 10% copper (ii) oxide... and convert that to a mass.

The TL;DR is that I don't think Bookshelf has the capability to do what I want around these blocks, and later when it drops items with data, it really won't help.

Reply:
What Bookshelf is doing right now is: Create an item which can be used to place a copper ore block, but which won't drop when you break it.

That's probably what you want. I'm just noting it's unexpected behaviour to anyone new to the codebase.